command/git.go

```diff
@@ -39,6 +44,7 @@ func (c *gitFetchProject) Name() string { return "git.get_project" }
 // ParseParams parses the command's configuration.
 // Fulfills the Command interface.
 func (c *gitFetchProject) ParseParams(params map[string]interface{}) error {
+	grip.Infof("%+v", params)
```

Comment:
should probably remove this, as it could leak tokens to the logs.

(in general, you logging things like this should have more context)